scripts/bootstrap.sh

```diff
@@ -38,11 +38,11 @@ _bootstrap_or_activate() {
 
     local _CHIP_BANNER="$(
         cat <<EOF
-  ▄███▒  ░▓█  ░▓█ ░▓█▓ ▒█████▄
- ██▒ ▀█▒  ▒█   ▒█  ░█▒  ▒█░  █░
- █▓░      ▒██████  ░█▒  ▒█▄▄▄█░
- ▓█   █▒  ▒█   ▒█  ░█░  ▒█▀
- ░▓███▀  ░▓███░▓█▒ ░█░  ▒█
+███    ███  █████  ████████ ████████ ███████ ██████  
+████  ████ ██   ██    ██       ██    ██      ██   ██ 
+██ ████ ██ ███████    ██       ██    █████   ██████  
+██  ██  ██ ██   ██    ██       ██    ██      ██   ██ 
+██      ██ ██   ██    ██       ██    ███████ ██   ██ 
```

Comment:
```suggestion
           ░▓░                                                                                      
           ▓█▓                                                                                      
           ▓█▓                                                     ▒█     ▒█                        
      ▒██▒▒▓██▒███▒                ░▒▓▒░  ░▒▓▒░       ░░▓█▒░ ░█  █████████████░    ░▒█▒░       ░░▒░ 
       ░▓█████▓██░               ▒█▒░░▒▓██▓▒░░▒█▒   ░█▓▒░░▒████    █▓░    █▓░    ▒█▒░░░▒█▓░   ██▒░░ 
    ▒█▒░         ░██▒           ░█░     █▓     ░█░ ░▓▒      ░▓█    █▓     █▓    ▒█░░    ░██  ▒▓     
    ░▓██▓░     ░██▓█░           ░█      ▓█      █░ ░█░       ██    █▓     █▓    ▓██████████  ▒█     
     ░▓███▒   ▒███▒░            ░█      ▓█      █░  ██░    ░███    █▓     █▓    ░█▒░         ▒█     
 ░▒████████░ ░███▓▓█▓▓▒         ░█      ▒▒      █░   ░█▓██▓█░▒█    ░▓▓█░  ░▓▓█░   ▒▓▓██▓█░   ▒▓     
 ░██▒░  ▒██▒ ▒██░  ░▒█▓                                                                             
         ░▓░ ░▓░   
```

Reply:
Ooh nice.

Reply:
Love the logo -- accepting.

Reply:
i did it quickly with an ascii art it is blurry can be better